obfx_modules/elementor-widgets/init.php

```diff
@@ -29,8 +29,21 @@ class Elementor_Widgets_OBFX_Module extends Orbit_Fox_Module_Abstract {
 	 */
 	public function __construct() {
 		parent::__construct();
-		$this->name           = __( 'Page builder widgets', 'themeisle-companion' );
-		$this->description    = __( 'Adds widgets to the most popular builders: Elementor or Beaver. More to come!', 'themeisle-companion' );
+		$this->name        = __( 'Page builder widgets', 'themeisle-companion' );
+		$this->description = __( 'Adds widgets to the most popular builders: Elementor or Beaver. More to come!', 'themeisle-companion' );
+
+		if ( wp_get_theme()->get( 'Name' ) === 'Neve' && ! is_plugin_active( 'neve-pro-addon/neve-pro-addon.php' ) ) {
+			$this->description .=
+				'<div class="neve-pro-notice">
+					<p>' /* translators: %s are bold html tags */
+						. sprintf( __( 'You can get access to %1$s10+ more Elementor and Gutenberg widgets%2$s, including Instagram integration, display conditions and more in %3$sNeve PRO%4$s.', 'themeisle-companion' ), '<b>', '</b>', '<b>', '</b>' ) . '
```

Comment:
@arinaturcu, I think it's better to combine the strings like this so when translation happens, they won't have parameters in the ending results that are replaced just by HTML tags. Also, the `Neve PRO` string doesn't need a translation.

```
sprintf(
		__( 'You can get access to %1$s, including Instagram integration, display conditions and more in %2$s.', 'themeisle-companion' ).
		sprintf( '<b>%1$s</b>',
			__( '10+ more Elementor and Gutenberg widgets', 'themeisle-companion' ),
		),
		'<b>Neve PRO</b>'
		);
```